tbx/project_styleguide/templates/patterns/_pattern_library_only/streamfield/work_story_container.html

```diff
@@ -0,0 +1,7 @@
+{% include "patterns/molecules/streamfield/blocks/stats_numeric.html" %}
```

Comment:
These don't show yet - I am hoping when the back-end is merged I can update the yaml, but as the back-end changes it I don't want to duplicate it yet.

Reply:
I've tried updating the yaml and the stats blocks don't display for some reason - I don't want to spend ages on it though.

Reply:
I went to look at this but can't see where to see the content in the Pattern Library. `work_story_container.html` is included by `stream_block_work.html` but viewing that template directly doesn't show anything and I can't see it being used by `work_page.html` - that looks to include `stream_block.html` which in turn includes `story_container.html`. I think I might have missed something?